test_main.py

```diff
@@ -0,0 +1,7 @@
+from pose_diff.Test.core.test_method import test_video
+
+def main():
+    test_video()
```

Comment:
Error occurs when moreInfo tab -> diffing is selected
Error Logs here
```vim
PermissionError: [WinError 32] 다른 프로세스가 파일을 사용 중이기 때문에 프로세스가 액세스 할 수 없습니다: 'temp\\output.avi'
```

Reply:
I think it's because writer of opencv hasn't been released.
I solve this problem.

Reply:
It seems to be solved. Thank you.